tests/sles4sap/monitoring_services.pm

```diff
@@ -29,7 +29,7 @@ sub configure_ha_exporter {
 
     # Get the IP port and start exporter
     my ($ha_exporter_port) = script_output("awk '/port:/ { print \$NF }' /etc/$exporter_name /usr/etc/$exporter_name 2>/dev/null", proceed_on_failure => 1) =~ /(\d+)/;
-    $ha_exporter_port ||= 9664;
+    $ha_exporter_port = ($ha_exporter_port =~ /^\d+$/ && $ha_exporter_port >= 0 && $ha_exporter_port <= 65536) ? $ha_exporter_port : 9664;
```

Comment:
What happens if wrong port number detected is between 0 and 65536? For example in https://openqa.suse.de/tests/11074867#step/monitoring_services/21 it detected 877.